ts/Core/Axis/Stacking/StackItem.ts

```diff
@@ -355,7 +355,9 @@ class StackItem {
             y = axis.toPixels(totalStackValue),
             xAxis = stackBoxProps.xAxis || chart.xAxis[0],
             x = pick(defaultX, xAxis.toPixels(this.x)) + xOffset,
-            yZero = axis.toPixels(boxBottom ? boxBottom : 0),
+            yZero = axis.toPixels(boxBottom ? boxBottom :
+                (axis.logarithmic && isNumber(axis.min) ?
+                    Math.pow(10, axis.min) : 0)),
```

Comment:
Use the `lin2log` function as usual. We can also do an _or_ statement. So something like

```ts
            yZero = axis.toPixels(
                boxBottom ||
                (isNumber(axis.min) && axis.logarithmic?.log2lin(axis.min)) ||
                0
            ),
            
```.